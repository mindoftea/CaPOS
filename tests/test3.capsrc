not 0x2acd9, 0, m2105
xor 0x164cd5, m2105, m2105
slr 0x1b75b5, m2105, m2105
slr m2105, 0x423d1, m2105
and 0x122e0e, 0x17cfa3, m219
add m219, 0x176243, m2105
not m2105, 0, m219
sub m2105, 0x675d1, m2105
or 0x17853e, 0xda8d8, m470
slr 0x18d601, m219, m219
and 0x3e26d, m219, m470
slr 0x86683, 0x1389ed, m2105
add m2105, 0x29285, m2105
not m2105, 0, m2105
sub 0x6eb4b, m470, m2105
and 0x17a930, 0x4e8aa, m219
not m470, 0, m2105
or 0x9f1c0, 0xb67d, m2105
or 0x173a42, 0x1bd5ad, m2905
slr 0xbbb55, 0x1ea20b, m470
or 0x1a65bd, 0xe00f0, m1716
or m470, 0x1ae883, m956
add 0x6f980, m956, m219
slr m956, 0x8db70, m219
slr 0x1c272d, 0xcc6dd, m2105
slr m1716, m956, m3745
and m956, 0x1dd072, m956
or m956, 0x18142, m956
xor 0x159538, 0xf2e04, m3745
and 0x87ab7, 0x2d334, m1716
not m1716, 0, m956
slr 0x101fb1, m470, m2570
add 0x5b306, 0x137d6b, m2570
add m956, 0x92638, m2905
slr 0xb585d, 0x147f59, m1716
slr m2905, m219, m2105
slr m470, 0x136682, m3745
add 0x14e4f5, m219, m3816
not m470, 0, m1716
or m2905, m1716, m1861
sub 0x1252d7, 0x1604ce, m181
sub m3745, 0x72c5a, m181
xor m956, 0x162519, m117
slr m2570, 0x12b74f, m2105
xor m1861, 0x1c12ce, m956
slr m181, m2905, m117
slr m181, 0x19f003, m181
not m2905, 0, m181
slr 0x9dcd7, m1861, m470
sub m3816, m1716, m956
add 0x1b5f30, 0x1f4c82, m1678
xor m2570, m219, m1716
print m1861, 0, 0
add 0x1016aa, m470, m1763
slr m1763, 0x86fd, m1678
slr m3745, 0x19d9cf, m3745
or m956, 0x107acc, m2014
and 0xd3f2e, m2105, m510
and m3745, m956, m465
sub 0x88fcf, m2570, m2721
xor m1763, 0x5a01e, m3745
not 0x485c6, 0, m1612
slr m2014, 0xdbacb, m510
sub m1763, m2014, m811
and 0x1cb8dd, m510, m465
xor 0x17f0e9, 0x1f262f, m2905
xor 0x25ab1, 0xbea83, m2129
xor 0x1e05cc, m2129, m2570
and m465, 0x17b7c1, m1716
not 0x1c0674, 0, m2129
sub m2014, 0x148a8d, m470
not m956, 0, m811
slr m956, m3816, m1861
or m219, 0x114abf, m956
print m2905, 0, 0
and 0xb382, 0x5bef1, m2905
sub 0x1499aa, m117, m956
or 0x614c5, 0xab1b, m470
sub m3816, m2905, m1678
print 0x1c894c, 0, 0
not m1612, 0, m1861
add 0xfb65d, m117, m2207
xor m956, 0xc7163, m2207
not m2905, 0, m2414
print 0xc8b3a, 0, 0
sub m1763, m510, m2791
and m2105, 0x67d52, m1716
slr 0x10395d, 0x24c3d, m117
sub m2791, m510, m2129
or 0x167c69, 0x21ef7, m811
sub m510, 0x532fa, m919
add m2105, m919, m2721
print m919, 0, 0
not 0x1ec739, 0, m2721
add m2570, m2129, m3816
slr 0x133826, m3745, m934
slr m117, m2414, m470
and m2014, m811, m2570
sub 0x9bdb, m465, m219
sub m919, 0x15bfea, m919
or 0xbe9af, 0x19fb8f, m181
slr m919, m2129, m181
sub m2014, 0x1b4726, m219
print 0xa1268, 0, 0
slr 0x28d44, m181, m2905
print m117, 0, 0
or m2105, 0x11836b, m2791
slr 0xdf962, m2570, m3948
or m1612, 0x66169, m181
xor m811, m2207, m2041
add m181, m3948, m2414
slr 0x1be361, 0x10c1bb, m3650
print m181, 0, 0
or 0x540d8, m2129, m3816
not m2721, 0, m1678
add 0xc8cc6, 0x1550fa, m3336
xor m510, 0x8bf68, m465
not m1612, 0, m510
xor 0xbe40, m465, m470
xor m2721, m1678, m219
print 0xc4c26, 0, 0
slr m1716, 0x292d, m2207
slr 0x1d1b8f, 0xa834d, m1763
xor 0x3a8d7, m2414, m3277
add m1612, m2414, m181
not 0x190ed8, 0, m2905
slr 0xb0cd0, m3816, m919
add 0x1b5a7c, m956, m811
xor m1716, 0x1ba7c9, m2640
slr m1678, 0x1923a2, m2570
add 0x20e5e, 0x1bd816, m2414
slr 0x1546e9, 0x98ad1, m470
not 0x7d037, 0, m3981
add m956, 0x1f6cf6, m2570
slr m510, m3336, m2721
or 0x15532c, 0x16ef4, m1716
slr m181, m2570, m983
sub 0x4e343, m3816, m2834
xor m2414, m2207, m3981
slr m2207, m1716, m2414
and m1716, 0x29853, m1712
or 0x1af96d, 0x1c45c7, m1678
not 0x1fb569, 0, m2640
sub 0x1e155e, m2014, m983
add 0x11de8a, 0x181f9b, m2129
add 0x183f8a, 0xd9406, m2105
and 0xb13f, m2105, m811
slr m470, 0xf09ea, m934
add m2721, 0x1957f0, m2014
slr m3745, 0x45c59, m1612
and m2640, m919, m2129
xor 0x73990, m510, m956
slr 0x19d5f0, m1763, m181
and m3336, 0x647d7, m219
slr 0x6a422, m3336, m1080
slr m811, 0x112554, m2721
add 0xd3036, m1612, m956
not 0x5d40, 0, m522
and 0x1b2be6, m1712, m3650
add m2207, m3816, m3816
add m2414, 0x174df5, m181
slr 0x5534, 0xb44bb, m181
slr 0x1746b0, m2834, m181
sub 0xdbf17, 0x1721b6, m1765
and 0x5486d, m1763, m811
print m2905, 0, 0
or 0xc1e90, 0xf0040, m3658
not 0x15d856, 0, m2538
or 0x1cc580, 0x1ba6f3, m1475
not 0xe9f24, 0, m2041
and m2041, m522, m1165
slr 0x15fb9e, m470, m3816
and m465, m3816, m956
xor m1678, m3650, m465
sub 0x1a0862, m2834, m2414
slr m1765, m2538, m3745
add m2129, m2905, m3658
xor 0x4581b, 0x18e2bc, m919
and 0xfd194, m1763, m1716
slr m2105, m470, m2834
add m117, m2834, m1763
and 0xc0a47, m1861, m3174
or 0x28227, 0x1ae320, m3745
or m3277, m1678, m3981
and m934, m1712, m510
and m956, m2834, m2871
not m2207, 0, m152
sub m3336, m1765, m2538
or 0x19389d, 0xc5d8b, m3545
sub m2570, m983, m2791
slr 0x131b0c, m2721, m2905
and m1612, m2791, m1678
slr 0xf211f, 0x112415, m3336
xor m1712, 0x1687ba, m859
slr m522, 0xff995, m2129
xor m2791, m1765, m3277
slr m1716, m2014, m2791
add 0x1382e5, 0x10d4a6, m36
print 0x1c9f01, 0, 0
and m470, 0x118369, m3650
print m2905, 0, 0
and 0x11c239, m3816, m510
add 0x12249c, 0x1ce57, m2207
xor 0x199595, m919, m117
or m919, 0x16d0f9, m1763
add m117, m2834, m2538
slr 0xf3d3e, 0x1f0582, m181
xor m2414, m2570, m3745
sub m1678, 0x6bda8, m2517
add 0x15ed40, 0xea003, m811
and 0x1a47a, 0x1fafe7, m181
sub 0x181e9d, m465, m181
or m219, 0x7e17, m2905
slr m2640, 0x3c198, m1765
sub m465, m510, m2014
not 0x169313, 0, m1424
or 0xcce21, m983, m2863
or m811, m2414, m1763
and m956, m3545, m3560
and m1678, m811, m1763
add 0xd87d5, m1080, m181
xor 0x62bdd, m2207, m2207
slr m1475, 0x3d625, m934
print m1475, 0, 0
sub m2517, 0x1dacbb, m1861
xor m3650, 0x59be4, m1424
or 0x4576d, 0x133415, m36
add 0x3281c, 0x19d584, m2570
not 0xe87b, 0, m1678
sub m3336, 0x8d9a3, m3650
slr m152, m181, m3650
add 0x4e6e5, m3948, m2014
sub m465, m983, m2414
add m2538, m152, m1738
slr 0xd0137, 0xe613a, m3545
add m2871, 0x172d8d, m181
slr m470, 0x1bf50f, m498
and 0x1ba1a7, m934, m2721
or m3545, m1424, m3277
not 0x73c57, 0, m2014
or m1678, 0x108002, m3948
slr m2414, 0x92a0e, m3336
sub m1612, m2905, m2115
sub 0x1bbff, 0x61f21, m2306
and 0xcff36, m2207, m633
add m510, 0x1f7fc0, m1080
slr 0x97025, m2905, m2517
or 0xc69ac, m522, m3650
sub m3277, 0x1b44c, m3703
sub m1165, m3277, m2791
print m1712, 0, 0
or 0x168cc2, 0x6f1ee, m3056
print 0x47f58, 0, 0
print m1861, 0, 0
not m919, 0, m2791
print m2640, 0, 0
