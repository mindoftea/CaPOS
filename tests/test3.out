0xfffffe8b
0x8d6c6
0x1c894c
0xc8b3a
0xfffacfa5
0xa1268
0x0
0xffff7b79
0xc4c26
0xffe6f127
0x1c9f01
0x131b0c
0x1fe7f3
0x1850
0x47f58
0xffdb959d
0xffe04a96
